<!DOCTYPE HTML>
<!--
	Minimaxing by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Tang Lab</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<!--[if lte IE 8]><script src="assets/js/ie/html5shiv.js"></script><![endif]-->
		<link rel="stylesheet" href="assets/css/main.css" />
		<!--[if lte IE 9]><link rel="stylesheet" href="assets/css/ie9.css" /><![endif]-->
	</head>
	<body>
		<div id="page-wrapper">
			<div id="header-wrapper">
				<div class="container">
					<div class="row">
						<div class="12u">

							<header id="header">
								<h1><a href="#" id="logo"><font style="text-transform: none;"> Tang Lab </font></a></h1>
								<nav id="nav">
									<a href="index.html" class="current-page-item"><font style="text-transform: none;"> Home </font></a>
									<a href="research.html"><font style="text-transform: none;"> Research </font></a>
									<a href="publications.html"><font style="text-transform: none;"> Publications </font></a>
									<a href="software.html"><font style="text-transform: none;"> Software </font></a>
									<a href="people.html"><font style="text-transform: none;"> People </font></a>
									<a href="jobs.html"><font style="text-transform: none;"> Jobs </font></a>
								</nav>
							</header>

						</div>
					</div>
				</div>
			</div>
			<div id="banner-wrapper">
				<div class="container">

					<div id="banner">
						<h2><font style="text-transform: none;"> Genomics and Metagenomics </font></h2>
						<span><font style="text-transform: none;"> Developing Statistical Methods and Computational Tools </font></span>
					</div>

				</div>
			</div>
			<div id="main">
				<div class="container">
					<div class="row main-row">
						<div class="12u 12u(mobile)">

							<section>
								<p> Our group focuses on the development and application of statistical methods and computational tools for high-dimensional "omics" data, arising from modern high-throughput technologies. Our current research interests include developing novel methods for integrating functional information in genetic association studies, meta-analysis for sequencing studies, and analysis of microbiome & metagenomic data.</p>
							</section>

						        <section>
							<h2>News</h2>
								<li>03/15/2023 Our PhyloMed paper has been accepted in <strong>Genome Biology</strong>. Good job, Kirin! PhyloMed is a method for microbiome mediation analysis. The method is implemented in the <a href="https://github.com/KiRinHong/miMediation">miMediation R package</a>.</li>	
								<br>
								<li>02/22/2023 Our MTAGEI paper has been accepted in <strong>Biostatistics</strong>. MTAGEI is a method for multi-trait analysis of gene-by-environment interactions in large-scale genetic studies. The R package is  <a href="https://github.com/lan/MTAGEI">here</a>.</li>	
								<br>
								<li>10/05/2022 Our team won Task 1 of the <a href="https://www.synapse.org/#!Synapse:syn26133770/wiki/">Preterm Birth Prediction - Microbiome DREAM Challenge</a>. Congrats to Jifan (PI: Guanhua Chen) and Zhoujingpeng! </li>
								<br>								
								<li>09/01/2021 We received funds from NSF (PI: Guanhua Chen) to develop methods for microbiome data. </li>
								<br>
								<li>08/20/2020 We received an R01 from NIGMS to develop methods for microbiome data. Looking forward to the fun research ahead of us!</li>
								<br>
								<li>07/27/2020 Our PSCAN paper has been accepted in <strong>Genome Biology</strong>. PSCAN is a protein-structure-guided scan method for testing gene-level associations and identifying risk variants on the protein space, available <a href="https://github.com/tangzheng1/PSCAN">here</a>.</li>	
								<br>
							        <li>05/09/2020 Lan's MTAR paper has been accepted in <strong>Nature Communications</strong>. Good job, Lan! MTAR is a method for multi-trait analysis of rare-variant association studies, available at: <a href="https://cran.r-project.org/web/packages/MTAR">https://cran.r-project.org/web/packages/MTAR</a>.</li>	
								<br>
								<li>02/27/2020 Lan Luo won ENAR and ASA Distinguished Student Paper Awards. Congrats, Lan!</li>
									
							
							</section>
							
						</div>


					</div>

				</div>
			</div>
			<div id="footer-wrapper">
				<div class="container">
					<h2>Contact Us</h2>
					
					<div class="row">
						
						<div class="8u 12u(mobile)">

							<section>
								<p>  Email: tang@biostat.wisc.edu   </p>
								
							</section>

						</div>
						<div class="4u 12u(mobile)">

							<section>
								<p>  </p>
							
							</section>

						</div>

					</div>
					<div class="row">
						<div class="12u">

							<div id="copyright">
								&copy; Tang Lab 2020. All rights reserved.
							</div>

						</div>
					</div>
				</div>
			</div>
		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/skel.min.js"></script>
			<script src="assets/js/skel-viewport.min.js"></script>
			<script src="assets/js/util.js"></script>
			<!--[if lte IE 8]><script src="assets/js/ie/respond.min.js"></script><![endif]-->
			<script src="assets/js/main.js"></script>

	</body>
</html>
