<!DOCTYPE HTML>
<!--
	Minimaxing by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title> Software - Tang Lab </title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<!--[if lte IE 8]><script src="assets/js/ie/html5shiv.js"></script><![endif]-->
		<link rel="stylesheet" href="assets/css/main.css" />
		<!--[if lte IE 9]><link rel="stylesheet" href="assets/css/ie9.css" /><![endif]-->
	</head>
	<body>
		<div id="page-wrapper">
			<div id="header-wrapper">
				<div class="container">
					<div class="row">
						<div class="12u">

							<header id="header">
								<h1><a href="#" id="logo"><font style="text-transform: none;"> Tang Lab </font></a></h1>
								<nav id="nav">
									<a href="index.html"><font style="text-transform: none;"> Home </font></a>
									<a href="research.html"><font style="text-transform: none;"> Research </font></a>
									<a href="publications.html"><font style="text-transform: none;"> Publications </font></a>
									<a href="software.html" class="current-page-item"><font style="text-transform: none;"> Software </font></a>
									<a href="people.html"><font style="text-transform: none;"> People </font></a>
									<a href="jobs.html"><font style="text-transform: none;"> Jobs </font></a>
								</nav>
							</header>

						</div>
					</div>
				</div>
			</div>
			<div id="main">
				<div class="container">
					<div class="row main-row">
						<div class="12u">

						<h2>Software</h2>
	
<section>
<p><strong><font size="5">MTAR: An R package to perform multi-trait analysis of rare-variant associations. </font></strong></p>
<p>MTAR is a framework for joint analysis of association summary statistics between multiple rare variants and different
traits, possibly from overlapping samples. MTAR tests leverage the genetic correlations to accommodate a wide variety of association patterns across traits and variants and enrich their associations.
Numerical study demonstrates that MTAR is substantially more powerful than single-trait-based tests and highlights the value of MTAR for novel pleiotropic gene discovery.</p>
<a href="https://github.com/tangzheng1/tanglab/blob/gh-pages/software/MTAR_v1.0.zip" class="button" download="download"><font style="text-transform: none;"> Download MTAR v1.0 </font></a>
</footer>
</section>
						
<section>
<p><strong><font size="5">PSCAN: An R package to perform protein-structure-guided scan tests for detecting gene-level associations and signal regions. </font></strong></p>
<p>PSCAN is a new set of gene-level association tests that leverage the tendency of functional variants to cluster in 3D protein space. 
PSCAN tests are built upon flexibly shaped spatial scan statistics, with scan windows adaptively defined to accommodate diverse topologies of variant positions in protein space.
PSCAN performs fast gene-level association tests by combining SNP-set-based testing p-values across windows using Cauchy method. 	
In addition, PSCAN implements a computationally efficient search algorithm for the detection of multiple
signal regions in protein space.</p>
<a href="https://uwmadison.box.com/s/198njl5v51g78021jroo04ywptuw1zsu" class="button" download="download"><font style="text-transform: none;"> Download PSCAN v1.0 </font></a>
</footer>
</section>
							
<section>
<p><strong><font size="5">miLineage: An R package to perform association tests for microbial lineages on a taxonomic tree. </font></strong></p>
<p>miLineage package has functions that implement a variety of association tests for microbiome data.
These functions allow users to (a) perform tests on multivariate taxon counts; (b) localize the covariate-associated lineages on the taxonomic tree; and (c) assess the overall association of the microbial community with the covariate of interest.</p>
<p><strong><font size="5">References:</font></strong><a href="https://academic.oup.com//bioinformatics/article/33/9/1278/2729907/A-general-framework-for-association-analysis-of?guestAccessKey=6c701407-b46c-49b1-a4d3-852545eeeead"> Tang ZZ <i>et al.</i> 2017 </a>, <a href="https://github.com/tangzheng1/tanglab/blob/gh-pages/publications/ZIGDM.pdf">Tang ZZ & Chen G 2018</a></p>
<footer class="controls">
<a href="https://github.com/tangzheng1/tanglab/blob/gh-pages/software/miLineage_v1.0.zip" class="button" download="download"><font style="text-transform: none;"> Download miLineage v1.0 </font></a>
<a href="https://cran.r-project.org/web/packages/miLineage/" class="button"><font style="text-transform: none;">miLineage v2.0 at CRAN</font></a>
<a href="https://github.com/tangzheng1/tanglab/blob/gh-pages/software/miLineage_v3.0.zip" class="button" download="download"><font style="text-transform: none;"> Download miLineage v3.0 </font></a>
</footer>
</section>
	
<section>
<p><strong><font size="5">miProfile: A C program for analyzing&nbsp;microbial community composition.</font></strong></p>
<p> miProfile is a command-line program written in the C language to implement the new distance-based method described in Tang et al. 2016a for performing association test for microbial community composition with any covariates of interest (e.g., environmental factors, disease status, clinical outcomes, treatment groups). miProfile can construct all commonly-used distances, including unweighted UniFrac, weighted UniFrac, generalized UniFrac, presence-weighted UniFrac, Bray-Curtis distance, and Jaccard distance. Users can request any combination of these distances. miProfile produces the p-value for each of the requested distances and the unified p-value by combining all of the distances. The operating interface is simple and can directly input the .biom and .tre files. The OTU abundance table and the distance matrices can be generated as output and re-used in subsequent runs. </p>
<p><strong><font size="5">References:</font></strong><a href="https://academic.oup.com/bioinformatics/article/32/17/2618/2450751/PERMANOVA-S-association-test-for-microbial?searchresult=1">Tang ZZ <i>et al.</i> 2016 </a>.</p>
<footer class="controls">
<a href="https://github.com/tangzheng1/tanglab/blob/gh-pages/software/miProfile_v1.0.zip" class="button" download="download"><font style="text-transform: none;"> Download miProfile v1.0 </font></a>
<a href="https://github.com/tangzheng1/tanglab/blob/gh-pages/software/miProfile_v2.0.zip" class="button" download="download"><font style="text-transform: none;"> Download miProfile v2.0 </font></a>
</footer>
</section>



<section>
<p><strong><font size="5">PreMeta: A C++ program to facilitate the exchange of information between four software packages for meta-analysis of rare-variant associations: MASS, RAREMETAL, MetaSKAT, and seqMeta.</font></strong></p>
<p> PreMeta is a software program written in C++ that is designed to facilitate the exchange of information between four software packages for meta-analysis of rare-variant associations: MASS, RAREMETAL, MetaSKAT, and seqMeta. PreMeta has two related purposes: one is to allow the use of different software packages within the same consortium; and the second is to eliminate the need to recalculate summary statistics when investigators join a new consortium that has adopted a different software package. </p>
<p><strong><font size="5">References:</font></strong><a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-3573-1">Tang ZZ <i>et al.</i> 2017</a>, <a href="http://www.sciencedirect.com/science/article/pii/S000292971500186X?via%3Dihub">Tang ZZ & Lin DY 2015</a></p>
<footer class="controls">
<a href="http://dlin.web.unc.edu/software/premeta/" class="button"><font style="text-transform: none;"> PreMeta Website </font></a>
</footer>
</section>
							

<section>
<p><strong><font size="5"> MASS: A C program for the meta-analysis of sequencing studies. </font></strong></p>
<p> MASS is a command-line program written in C to perform fixed-effects (FE) and random-effects (RE) meta-analysis of sequencing studies by combining the score statistics from multiple studies. It implements three types of tests that encompass all commonly used association tests for rare variants, including simple burden test, CMC test (Li and Leal, 2008), weighted sum statistic (Madsen and Browning, 2009), variable-threshold (VT) test (Price et al., 2010; Lin and Tang, 2011), C-alpha test (Neale et al., 2011) and SKAT (Wu et al., 2011). The input file can be generated from the accompanying software SCORE-Seq. This bundle of programs allows meta-analysis of sequencing studies in a statistically accurate, numerically stable and computationally efficient manner. </p>
<p><strong><font size="5">References:</font></strong><a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btt280">Tang ZZ & Lin DY 2013</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/24799183">Tang ZZ & Lin DY 2014</a></p>
<footer class="controls">
<a href="http://dlin.web.unc.edu/software/mass/" class="button"><font style="text-transform: none;"> MASS Website </font></a>
</footer>
</section>
							


<section>
<p><strong><font size="5"> SCORE-SeqTDS: A C program for analyzing primary and secondary traits in sequencing studies under trait- dependent sampling. </font></strong></p>
<p> SCORE-SeqTDS is a software program developed for analyzing primary and secondary quantitative traits under trait-dependent sampling. The primary trait is the trait that is used to select subjects for sequencing, and all other traits are treated as secondary. Each quantitative trait is related to a genetic variable and possibly covariates through a linear regression model. Both the maximum likelihood estimation (MLE) and standard least-squares (LS) methods are available. The MLE method properly accounts for trait-dependent sampling whereas the LS method does not. The LS method is the ideal choice for random sampling and is approximately correct for analyzing secondary quantitative traits in case-control or case-only studies with rare diseases. SCORE-SeqTDS performs the LS analysis on secondary quantitative traits for random sampling, case-control and case-only sampling. For random sampling, all traits are treated as secondary (because the sampling does not depend on any particular trait.) </p>
<p><strong><font size="5">References:</font></strong><a href="http://www.pnas.org/content/110/30/12247.full"> Lin DY, Zeng D, & Tang ZZ 2013</a>.</p>
<footer class="controls">
<a href="http://dlin.web.unc.edu/software/score-seqtds/" class="button"><font style="text-transform: none;"> SCORE-SeqTDS Website </font></a>
</footer>
</section>
		


<section>
<p><strong><font size="5"> SCORE-Seq: A C program which implements score statistics for detecting disease associations with rare variants in sequencing studies. </font></strong></p>
<p> SCORE-Seq is a command-line program which implements score statistics for detecting disease associations with rare variants in sequencing studies. The mutation information is aggregated across multiple variant sites of a gene through a weighted linear combination and then related to disease phenotypes through appropriate regression models. The weights can be constant or dependent on allele frequencies and phenotypes. The association testing is based on score statistics. The allele-frequency threshold can be fixed or variable. Statistical significance can be assessed by using asymptotic normal approximation or resampling. The current release covers binary and continuous traits with arbitrary covariates under case-control and cross-sectional sampling. </p>
<p><strong><font size="5">References:</font></strong><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3169821/"> Lin DY & Tang ZZ 2011</a>.</p>
<footer class="controls">
<a href="http://dlin.web.unc.edu/software/score-seq/" class="button"><font style="text-transform: none;"> SCORE-Seq Website </font></a>
</footer>
</section>


<section>
<p><strong><font size="5"> genoCN: An R package for identifying copy number states and genotype calls using the SNP arrays data. </font></strong></p>
<p> GenoCN is a software that simultaneously identify copy number states and genotype calls. Different strategies are implemented for the study of Copy Number Variations (CNVs) and Copy Number Aberrations (CNAs). While CNVs are naturally occurring and inheritable, CNAs are acquired somatic alterations most often observed in tumor tissues only. CNVs tend to be short and more sparsely located in the genome compared to CNAs. GenoCN consists of two components, genoCNV and genoCNA, designed for CNV and CNA studies, respectively. In contrast to most existing methods, genoCN is more flexible in that the model parameters are estimated from the data instead of being decided a priori. genoCNA also incorporates two important strategies for CNA studies. First, the effects of tissue contamination are explicitly modeled. Second, if SNP arrays are performed for both tumor and normal tissues of one individual, the genotype calls from normal tissue are used to study CNAs in tumor tissue. </p>
<p><strong><font size="5">References:</font></strong><a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkp493"> Sun <i> et al.</i> 2009</a>.</p>
<footer class="controls">
<a href="https://www.bioconductor.org/packages/release/bioc/html/genoCN.html" class="button"><font style="text-transform: none;"> genoCN Website </font></a>
</footer>
</section>


							

						</div>
					</div>
				</div>
			</div>
			<div id="footer-wrapper">
				<div class="container">
					<h2>Contact Us</h2>
					
					<div class="row">
						
						<div class="8u 12u(mobile)">

							<section>
								<p> Mailing address: 330 N Orchard St, Madison, WI 53715, United States </p>
								<p>	Office location: WID 3166 </p>
							</section>

						</div>
						<div class="4u 12u(mobile)">

							<section>
								<p> Email: tang@biostat.wisc.edu </p>
								<p>	Office phone: (608) 316-4643 </p>
							</section>

						</div>

					</div>
					<div class="row">
						<div class="12u">

							<div id="copyright">
								&copy; Tang Lab 2020. All rights reserved.
							</div>

						</div>
					</div>
				</div>
			</div>
		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/skel.min.js"></script>
			<script src="assets/js/skel-viewport.min.js"></script>
			<script src="assets/js/util.js"></script>
			<!--[if lte IE 8]><script src="assets/js/ie/respond.min.js"></script><![endif]-->
			<script src="assets/js/main.js"></script>

	</body>
</html>
